polkadot/primitives/src/v5/mod.rs

```diff
@@ -830,60 +830,6 @@ pub struct ParathreadEntry {
 	pub retries: u32,
 }
 
-/// An assignment for a parachain scheduled to be backed and included in a relay chain block.
-#[derive(Clone, Encode, Decode, PartialEq, TypeInfo, RuntimeDebug)]
-pub struct Assignment {
-	/// Assignment's ParaId
-	pub para_id: Id,
-}
-
-impl Assignment {
-	/// Create a new `Assignment`.
-	pub fn new(para_id: Id) -> Self {
-		Self { para_id }
-	}
-}
-
-/// An entry tracking a paras
-#[derive(Clone, Encode, Decode, TypeInfo, PartialEq, RuntimeDebug)]
```

Comment:
These types are only used internally, no need to expose them via primitives.

Reply:
Wonderful. Thanks for refactoring this and reducing the API surface.